routing/edge_estimator.cpp

```diff
@@ -17,33 +32,60 @@ inline double TimeBetweenSec(m2::PointD const & from, m2::PointD const & to, dou
 class CarEdgeEstimator : public EdgeEstimator
 {
 public:
-  CarEdgeEstimator(IVehicleModel const & vehicleModel);
+  CarEdgeEstimator(IVehicleModel const & vehicleModel, traffic::TrafficCache const & getter);
```

Comment:
I left name getter here because class TrafficCache is used as a getter in CarEdgeEstimator.